views/movies/movie-details.hbs

```diff
@@ -0,0 +1,16 @@
+<h2>Movie Details</h2>
+
+
+<h3></h3>{{this.movieDetails.title}}</h3>
+<p>Genre: {{this.movieDetails.genre}}</p>
+<p>Plot: {{this.movieDetails.plot}}</p>
+
+{{#each movieDetails}}
+<p>{{this.cast}}</p>
+<p>{{this.cast.occupation}}</p>
+<p>{{this.cast.catchPhrase}}</p>
+{{/each}}
+
+<form action="/movies/{{this._id}}/delete" method="POST">
+<Button>Delete movie</Button>
+</form>
```

Comment:
- When I try to delete a movie, I get 404 -  route does not exist.
- Our URL looks like this `http://localhost:3000/movies//delete` - as you can see ID is not being passed - and this points where to look for error
- When you display movie details, stay consistent. When you print title, genre etc, you write `this.movieDetails.title`, however when you print id. you use this syntax `_id` - event hough we're accessing exact same object ;)